04-django/01-python-intro/greet.py

```diff
@@ -0,0 +1,3 @@
+def greeting(name):
+    print('Hello '+ name + '!')
```

Comment:
or use f-Strings like this: `f'Hello {name}'`

Reply:
Will try it when it's more complicated. Typing less is cool.